package.json

```diff
@@ -58,6 +58,7 @@
 		"@wordpress/element": "^5.27.0",
 		"@wordpress/react-i18n": "^3.48.0",
 		"autoprefixer": "^10.4.16",
+		"copy-webpack-plugin": "^12.0.2",
```

Comment:
I added this plugin as the about menu is in the HTML file and this seems to be a common practice for copying the files that don't need to be processed by webpack